src/sentry/api/serializers/models/organization.py

```diff
@@ -143,6 +143,14 @@ def serialize(self, obj, attrs, user):
         if features.has('organizations:event-attachments', obj, actor=user):
             feature_list.append('event-attachments')
 
+        # this is slightly gross but necessary until we have the get on the endpoint
```

Comment:
can we just delete the feature handler and use the experiment directly here?

Reply:
oh forgot this was in sentry. I'm not sure what this comment means, can you say something like "the experiment definition is in getsentry and the features framework provides a convenient way to invoke it from there and send it over to the frontend"